tests/unit/data/test_inference.py

```diff
@@ -9,7 +9,7 @@
 import albumentations as A  # noqa: N812
 import pytest
 
-from anomalib.data.inference import InferenceDataset
+from anomalib.data.inference import PredictDataset
```

Comment:
is this correct? shouldn't be `anomalib.data.predict import PredictDataset`? 
Alternatively, we could just do `from anomalib.data import PredictDataset` to be on the safe side?